pkgs/development/compilers/elm/default.nix

```diff
@@ -31,6 +36,11 @@ let
             */
             elm-format = justStaticExecutables (overrideCabal (self.callPackage ./packages/elm-format.nix {}) (drv: {
               jailbreak = true;
+
+              description = "elm-format formats Elm source code according to a standard set of rules based on the official Elm Style Guide";
+              homepage = "https://github.com/avh4/elm-format";
+              license = licenses.bsd3;
+              maintainers = [ maintainers.avh4 maintainers.turbomack ];
```

Comment:
@avh4 just so you know - I'm adding you as a maintainer to elm-format. I believe in past you were declared directly in `elm-format.nix` file but I think it was overwritten by cabal2nix when the file was regenerated during some update. Defining this here means we can regenerate the file without loosing the information. Feel free to suggest change if you don't want to be declared as a maintainer anymore.

Reply:
Update: turned out your profile is not in maintainer list so I removed you.